src/bot.py

```diff
@@ -142,3 +202,53 @@ async def load_passwords(self) -> AsyncGenerator[Tuple[str, str], None]:
                 content: str = message.content.replace('`', '').strip()
                 team, password = content.split(':')
                 yield team.strip(), password.strip()
+
+    def stats_message(self, members: bool = True, warnings: bool = True, statistics: bool = False) -> str:
+        """Generate a message string for the given options."""
+        return '\n\n'.join([
+            *([self.teams_data.team_summary()] if members else []),
+            *([self.teams_data.warnings()] if warnings else []),
+            *([self.teams_data.statistics()] if statistics else []),
+        ])
+
+    def add_subscribed_message(self, msg: SubscribedMessage) -> None:
+        """Add a subscribed message to the subscribed list."""
+        self.subscribed_messages.append(msg)
+        self._save_subscribed_messages()
+
+    async def remove_subscribed_message(self, msg: SubscribedMessage) -> None:
+        """Remove a subscribed message from the channel and subscribed list."""
+        msg_channel = await self.fetch_channel(msg.channel_id)
+        if not hasattr(msg_channel, 'fetch_message'):
+            # ignore for channels that don't support message editing
+            return
+        message = await msg_channel.fetch_message(msg.message_id)
```

Comment:
You probably need some handling here to cope with the message not existing in case an admin has manually deleted the message.

Reply:
b3939af